com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -60,15 +62,17 @@ public static bool Initialize()
 #if UNITY_EDITOR || !UNITY_ANDROID
             s_NotificationManager = null;
             s_Initialized = false;
+            s_NotificationManagerClass = null;
+            s_NotificationManagerContext = null;
 #elif UNITY_ANDROID
             AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
             AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
```

Comment:
Do you think if we should do the same to these two variables(converting to s_***)? As we use them in GetLastNotificationIntent().

Reply:
Good idea, thanks